content/de/docs/concepts/workloads/pods/_index.md

```diff
@@ -0,0 +1,373 @@
+Container direkt zu verwalten.
+* **Pods, in denen mehrere Container ausgeführt werden, die zusammenarbeiten
+müssen**. Wenn eine Softwareanwendung aus co-lokaliserten Containern besteht, 
+die sich gemeinsame Ressourcen teilen und stark voneinander abhängen, kann ein 
+Pod die Container verkapseln. 
+Diese Container bilden eine einzelne zusammenhängende 
+Serviceeinheit, z. B. ein Container, der Daten in einem gemeinsam genutzten 
+Volume öffentlich verfügbar macht, während ein separater _Sidecar_-Container 
+die Daten aktualisiert. Der Pod fasst die Container, die Speicherressourcen
+und eine kurzlebiges Netzwerk-Identität als eine Einheit zusammen.
+
+{{<note>}}
+Das Gruppieren mehrerer gemeinsam lokalisierter und gemeinsam verwalteter
+Container in einem einzigen Pod ist ein relativ fortgeschrittener 
+Anwendungsfall. Sie sollten diese Architektur nur in bestimmten  Fällen 
+verwenden, wenn Ihre Container stark voneinander abhängen.
+{{</ note>}}
+
+Jeder Pod sollte eine einzelne Instanz einer gegebenen Anwendung ausführen. Wenn 
+Sie Ihre Anwendung horizontal skalieren wollen (um mehr Instanzen auszuführen
+und  dadurch mehr Gesamtressourcen bereitstellen), sollten Sie mehrere Pods 
+verwenden, 
+einen für jede Instanz. In Kubernetes wird dies typischerweise als Replikation
+bezeichnet.
+Replizierte Pods werden normalerweise als eine Gruppe durch eine 
+Workload-Ressource und deren 
+{{<glossary_tooltip text = "Controller" term_id = "controller">}} erstellt 
+und verwaltet.
+
+Die Seite [Pods und Controller](#pods-and-controllers) beschreibt, wie 
+Kubernetes Workload-Ressourcen und deren Controller verwendet, um Anwendungen 
+zu skalieren und zu heilen.
+
+### Wie Pods mehrere Container verwalten
+
+Pods unterstützen mehrere kooperierende Prozesse (als Container), die eine 
+zusammenhängende Serviceeinheit bilden. Kubernetes plant und stellt automatisch 
+sicher, dass sich die Container in einem Pod  auf demselben physischen oder 
+virtuellen Server im Cluster befinden. Die Container können Ressourcen und
+Abhängigkeiten gemeinsam nutzen, miteinander kommunizieren und
+ferner koordinieren wann und wie sie beendet werden.
+
+Zum Beispiel könnten Sie einen Container haben, der als Webserver für Dateien in
+einem gemeinsamen Volume arbeitet. Und ein separater "Sidecar" -Container 
+aktualisiert die Daten von einer externen Datenquelle, siehe folgenden 
+Abbildung:
+
+{{< figure src="/images/docs/pod.svg" alt="example pod diagram" width="50%" >}}
+
+Einige Pods haben sowohl {{<glossary_tooltip text = "Initialisierungs-Container" 
+term_id = "init-container">}} als auch {{<glossary_tooltip text = 
+"Anwendungs-Container" term_id = "app-container">}}. 
+Initialisierungs-Container werden gestartet und beendet bevor die 
+Anwendungs-Container gestartet werden.
+
+Pods stellen standardmäßig zwei Arten von gemeinsam Ressourcen für die 
+enthaltenen Container bereit:
+[Netzwerk](#pod-networking) und [Speicher](#pod-storage).
```

Comment:
These links don't match the fragment identifiers in the rendered page. Try it at https://deploy-preview-26192--kubernetes-io-master-staging.netlify.app/de/docs/concepts/workloads/pods/